specs/configuration_files_specification.md

```diff
@@ -88,4 +91,5 @@ be supported by implementations. In such schemes many drop-ins are loaded from a
 directory in each hierarchy.
 
 For example, `/usr/lib/foo.d/a.conf`, `/usr/lib/foo.d/b.conf` and `/etc/foo.d/c.conf`
-are all loaded and parsed in this scheme.
+are all loaded and parsed in this scheme,
+in this order.
```

Comment:
this line break looks a bit too eager? just a nit

Reply:
I removed the line break.